src/XRP/issued-currency-client.ts

```diff
@@ -505,6 +505,32 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Disables rippling on the trust line between this account (issuing account) and another account.
+   * Note that the trust line's limit is set to 0.
```

Comment:
Is this true??  I don't see this noted in the documentation, and the final assert statement in your test suggests otherwise.

Reply:
it's false, deleted